src/main/scala/scalaz/analytics/AnalyticsModule.scala

```diff
@@ -123,24 +144,55 @@ trait AnalyticsModule {
     def + (r: A =>: B)(implicit B: Numeric[B]): A =>: B = (l &&& r) >>> B.sum
     def - (r: A =>: B)(implicit B: Numeric[B]): A =>: B = (l &&& r) >>> B.diff
     def % (r: A =>: B)(implicit B: Numeric[B]): A =>: B = (l &&& r) >>> B.mod
+
+    def > (r: A =>: B)(implicit B: Numeric[B]): A =>: Boolean = (l &&& r) >>> B.greaterThan
```

Comment:
Should this be on `Numeric` or a more general `Comparable` typeclass of some sort?